### ISSUE TITLE
WCAG 1.3.3 Sensory Characteristics G96 todo

### ISSUE THREAD

**Issue by kimpham54** [dulibrarytech/digitaldu-frontend#126]
G96: Providing textual identification of items that otherwise rely only on sensory information to be understood LTS Note: logos or shapes should have text alternatives.
